coral/src/app/hooks/useFeatureFlag.ts

```diff
@@ -0,0 +1,16 @@
+import { useState, useEffect } from "react";
+
+enum FeatureFlag {
+  TOPIC_REQUEST = "FEATURE_FLAG_TOPIC_REQUEST",
+}
+
+const useFeatureFlag = (flagName: FeatureFlag) => {
```

Comment:
Maybe we could add some documentation about this feature flag feature?

Reply:
Where would you add the documentation? As a [jsdoc](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html) or a separate document? I think you have already learned I am poor documenter.

Reply:
I would say a new MD file here named "Feature flags" https://github.com/aiven/klaw/tree/main/coral/docs

And maybe a paragraph here to explain to new contributors to coral that they should hide WIP features merged behind a feature flag, and a link to the previously mentioned MD file: https://github.com/aiven/klaw/blob/main/CONTRIBUTING.md

Reply:
Excellent, thank you ^^